### PR TITLE
add hazesoft

### PR DESCRIPTION
**Pull request by gaurovgiri**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [X] Added new company
- [ ] Other(s)

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -368,6 +368,22 @@
         "youtube": "https://www.youtube.com/@Hamropatro"
       }
     },
+    {
+      "name": "HazeSoft Pvt. Ltd.",
+      "establishedAt": 1350227551,
+      "location": "Shankhamul-31, Kathmandu, Nepal",
+      "website": "https://hazesoft.co/",
+      "email": "info@hazesoft.co",
+      "phone": "01-4787293",
```
Comment (thenishantsapkota) [inline data/companies.json:377]:
```suggestion
      "phone": "+977-14787293",
```

File: data/companies.json [hunk 2]
```diff
@@ -368,6 +368,22 @@
         "youtube": "https://www.youtube.com/@Hamropatro"
       }
     },
+    {
+      "name": "HazeSoft Pvt. Ltd.",
+      "establishedAt": 1350227551,
```
Comment (thenishantsapkota) [inline data/companies.json:373]: What is the source for this?

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/companies.json (hunk 2 above)
Comment (gaurovgiri) [reply data/companies.json:373]: image

**Review by thenishantsapkota — APPROVED** [review]
LGTM!
Thank you for your contribution @gaurovgiri!

**Pull request merged**